src/cdk/a11y/key-manager/list-key-manager.spec.ts

```diff
@@ -480,6 +480,41 @@ describe('Key managers', () => {
       });
     });
 
+    describe('skip predicate', () => {
+
+      it('should skip disabled items by default', () => {
+        itemList.items[1].disabled = true;
+
+        expect(keyManager.activeItemIndex).toBe(0);
+
+        keyManager.onKeydown(fakeKeyEvents.downArrow);
+
+        expect(keyManager.activeItemIndex).toBe(2);
+      });
+
+      it('should be able to not skip any item', () => {
+        keyManager.skipPredicate(() => false);
+
+        itemList.items.forEach(item => item.disabled = true);
+
+        keyManager.onKeydown(fakeKeyEvents.downArrow);
+
+        expect(keyManager.activeItemIndex).toBe(1);
+      });
+
+      it('should be able to skip items with a custom predicate', () => {
+        keyManager.skipPredicate((item: any) => item.skipItem);
```

Comment:
I think the `item` here is a `FakeFocusable`. Alternatively you could let TS infer the type itself.

Reply:
:+1: